src/components/Button/Button.types.ts

```diff
@@ -118,6 +114,10 @@ export interface ButtonProps extends NativeButtonProps {
      * The button class names.
      */
     classNames?: string;
+    /**
+     * The button counter number.
+     */
+    counter?: number;
```

Comment:
can we change this to type string for large number use cases. For eg: we might `3k` instead of `3000`